content/300-guides/200-deployment/301-edge/450-deploy-to-cloudflare-workers.mdx

```diff
@@ -155,146 +129,149 @@ enum Level {
+## 4. Enable Accelerate in the Prisma Data Platform
 
-   </Admonition>
+Prisma currently does not work on Cloudflare Workers yet. However, you can use Prisma on Cloudflare Workers through [Prisma Accelerate](/data-platform/accelerate).
 
-   ![Signup for Prisma Data Platform](../../../doc-images/data-platform/pdp-signup.png)
+To get started with Prisma Accelerate:
 
-2. Click **New Project** and then select **Import a Prisma repository**.
-3. Fill in the repository and project details, and click **Next**.
+1. Sign up for a free [Prisma Data Platform account](https://console.prisma.io/)
+1. Create a project
+1. Navigate to the project you created
+1. Enable Accelerate
+1. Generate an Accelerate connection string and copy it to your clipboard
 
-   ![Import a Project](./images/450-03-import-project.png)
+## 5. Configure the Accelerate connection string in your project
 
-4. Next, you connect the Prisma Data Platform to the MongoDB Atlas database and set up the Data Proxy.
+1. Rename the existing `DATABASE_URL` environment variable to `DIRECT_URL`. The `DIRECT_URL` variable will be used perform migrations and introspections.
+1. Add the Prisma Accelerate connection string to your `.env` file.
 
-   1. Paste the MongoDB Atlas connection string which also includes the database name as described in [1. Get a MongoDB Atlas connection string](#1-get-a-mongodb-atlas-connection-string).
-   2. Under **Location**, select a Data Proxy location that is geographically close to your MongoDB database location.
-   3. Enable **Static IPs** and copy and paste them in the _Network Access_ screen in MongoDB Atlas.
-
-      ![Connect your Database](./images/450-04-connect-db.png)
-
-   4. Click **Create Project** to test the connection and set up the Data Proxy.
-
-      <Admonition type="warning">
-
-      If you see, "The database needs to be empty to proceed", you can simply use a different database name. Using the screenshot above, that would be renaming `Log` to something else.
-
-      </Admonition>
-
-      If all goes well, you'll be greeted with a new connection string that starts with `prisma://`.
-
-   5. Copy the `prisma://` connection string to your clipboard.
+   ```diff file=.env
+   -DATABASE_URL="postgres://..."
+   +DATABASE_URL="prisma://accelerate.prisma-data.net/?api_key=__API_KEY__"
+   +DIRECT_URL="postgres://..."
+   ```
 
-      ![Data Proxy Page](./images/450-05-data-proxy.png)
+   Add [`directUrl`](/reference/api-reference/prisma-schema-reference#fields) property in `datasource` block in the `schema.prisma` file.
 
-## 7. Set the Data Proxy Connection string in your environment
+```diff file=prisma/schema.prisma
+datasource db {
+provider = "postgresql"
+url = env("DATABASE_URL")
++directUrl = env("DIRECT_URL")
+}
+```
 
-1. Add the Data Proxy connection string to your local environment `.env` file.
+> The `directUrl` field is not required for Prisma Accelerate. It allows you to easily perform database migrations and introspections.
```

Comment:
As usual, indentation of this block is broken in the list:
<img width="546" alt="image" src="https://github.com/prisma/docs/assets/183673/a1b85ebf-769d-468f-9bb0-f2f034de27bd">

Reply:
This is tricky to navigate. If I indent the block, the diff highlighting will not work. I'll try find a way around this

Reply:
I've resorted to using the `prisma` language and adding a highlight instead. That should resolve it.